bergson/gradients.py

```diff
@@ -465,10 +484,14 @@ def _save_input(self, module: nn.Module, inp: tuple, _):
 
             x = x * b.type_as(x)  # [N, S, I] * [I] → [N, S, I]
 
+        include_bias = (
+            getattr(module, "bias", None) is not None and self.processor.include_bias
+        )
+
         # If we're not using AdamNormalizer, we can randomly project the input here
         # to save memory, rather than waiting until the backward pass.
```

Comment:
Update this comment to explain the reasoning behind the bias handling here (i.e. different gradient calculations being concatenated)